### ISSUE TITLE
Unable to get resolved name for use clause name

### ISSUE THREAD

**Issue by dantleech** [Microsoft/tolerant-php-parser#164]
Getting the resolved name of a use clause using `QualifiedName#getResolvedName` is [explicitly disallowed](https://github.com/Microsoft/tolerant-php-parser/blob/master/src/Node/QualifiedName.php#L87) and I am not sure why, it seems legitimate to me?

```php
namespace SomeNamespace;

use Something\HelloTrait;

class Foobar
{
    use HelloTrait;
}
```
```
$name->getResolvedName()
```

Expected: `Something\HelloTrait`
Actual: `null`.

**Comment by roblourens** [comment]
I don't have a good answer... it seems like they should be resolved in a similar way to usages of the names. I'm not sure why that limitation is in place, but that API is used in a lot of places, so I'll have to think about it some more.

**Comment by mousetraps** [comment]
Good catch. Honestly, based on the blame - it looks like a hack that inadvertently got committed from when I was working on completions support. So removing it might break something with completions, but that something should probably be fixed in other ways :wink:
